packages/react-server/src/ReactFlightServer.js

```diff
@@ -4094,3 +4090,8 @@ export function halt(request: Request, reason: mixed): void {
     fatalError(request, error);
   }
 }
+
+function allReady(request: Request) {
+  const onAllReady = request.onAllReady;
+  onAllReady();
+}
```

Comment:
If we unify the rest of halt/abort, then this helper doesn't become necessary.

Reply:
I'll do in the next where we re-add the error and postpone logging